geometry_msgs/Vector3 pos
geometry_msgs/Vector3 rpy
geometry_msgs/Vector3 lin_vel
float64 MSE
float64 MSE_pos
float64 MSE_vel
float64 MSE_rpy
